plugins/file/index.js

```diff
@@ -69,6 +70,8 @@ module.exports = {
       },
       handler: async function(request, h) {
         const file = request.payload.file;
+        const fileKey = request.payload.fullPath;
```

Comment:
This has to be `let fileKey` because it gets redefined later in the code if `fileKey` is undefined